spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -336,5 +336,20 @@ class SpdxExpressionTest : WordSpec() {
                 spdxExpression.validChoices() shouldContainExactlyInAnyOrder choices
             }
         }
+
+        "offers choice" should {
```

Comment:
Again, use the function name? Or "Offers choice" (capitalized)?

Reply:
Done.